Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -346,6 +351,26 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        self.preferenceHelper.universalLinkUrl = [userActivity.webpageURL absoluteString];
```

Comment:
crudely handled now by returning NO if `bnc.lt` isn't present in the `webpageURL`